src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCtorCall.java

```diff
@@ -0,0 +1,72 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+/**                                                                           //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:   //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ * basicOffset = 2                                                            //indent:1 exp:1
+ * braceAdjustment = 0                                                        //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                //indent:1 exp:1
+ * tabWidth = 4                                                               //indent:1 exp:1
+ */                                                                           //indent:1 exp:1
+class InputIndentationCtorCall { //indent:0 exp:0
```

Comment:
issue is "lambda as the first parameter in call base class ctor", here are lambda in user defined test case.
but here is no "->" in input file...
1) As good practice ... reported test case is copied "as is" to Inputs to make sure it is covered for sure.
2) if smth else if going to be fixed (not releated to referenced issue, it is better to create new issue.)

Reply:
This issue is not related to lambda. As Richard said, the problem is that the call to base constructor is simply ignored by the indentation check.
But you are right, the harder the tests, the more they will cover.

Reply:
@romani Issue title was user made and they didn't know the underlying issue.
I have corrected the issue title. Feel free to modify it if you need more.

Reply:
:) you mentioned it in title then in description

>Only happends with 'super' and only if the lambda comes first.

You did mention unrelated stuff too much.
That is why it is good practice to use test case from user as they do not know details, and just report what whey see.

Please fix description if it second "typo".

Reply:
> You did mention unrelated stuff too much.

@romani If you are referring to me, I only changed the title, not the description of the issue.

Reply:
node. @romani are you waiting from me to improve anything else for this PR?